ansible/sf_snapshot.py

```diff
@@ -55,6 +55,8 @@ def present(module):
         extra += ' --all'
     if module.params.get('label'):
         extra += ' --label_name %s' % module.params['label']
+    if module.params.get('delete_after_label') and module.params['delete_after_label']:
```

Comment:
Does this line need the `and module.params['delete_after_label']`?

Reply:
Ummm, probably not. Its the pattern we've used elsewhere. I'll fix.